app/src/backend/CalculationService.ts

```diff
@@ -73,70 +81,193 @@ export default class CalculationService {
     const gasToCO2Eqs = await db.models.GasToCO2Eq.findAll();
     let totalCO2e = 0n;
     let totalCO2eYears = 0;
-    let gases: { gas: string; amount: bigint }[] = [];
+    let gases: Gas[] = [];
+
     switch (formula) {
+      // TODO use Record<string, (activityValue, gasToCO2Eqs) => FormulaResult> for this? To avoid adding new code here for each new formula...
+      // basically like a function pointer table in C++...
       case "direct-measure":
```

Comment:
This will be resolved during the other refactoring (as they will be the keys of a constant object, which we can then also export as constants if we want.